bokehjs/src/lib/models/filters/boolean_filter.ts

```diff
@@ -33,18 +33,18 @@ export class BooleanFilter extends Filter {
     if (booleans != null && booleans.length > 0) {
       if (every(booleans, isBoolean)) {
         if (booleans.length !== source.get_length()) {
-          logger.warn(`BooleanFilter ${this.id}: length of booleans doesn't match data source`)
+          logger.warn(`${this}: length of booleans doesn't match data source`)
         }
         return range(0, booleans.length).filter((i) => booleans[i] === true)
       } else {
-        logger.warn(`BooleanFilter ${this.id}: booleans should be array of booleans, defaulting to no filtering`)
+        logger.warn(`${this}: booleans should be array of booleans, defaulting to no filtering`)
         return null
       }
     } else {
       if (booleans != null && booleans.length == 0)
-        logger.warn(`BooleanFilter ${this.id}: booleans is empty, defaulting to no filtering`)
+        logger.warn(`${this}: booleans is empty, defaulting to no filtering`)
```

Comment:
Does this need a warning at all? Setting the filter to empty to turn off filtering seems pretty standard/comprehensible, and wanting to turn off filtering at times seems reasonable.

Reply:
I suppose not. In fact I've looking forward to refactoring filters for a while. In any case, that's an orthogonal concern I can address in another PR.

Reply:
Sure, can you make an issue for the work you'd like to do so it can be visible in a millstone at some point?